splice_machine/pom.xml

```diff
@@ -165,6 +165,22 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka_${scala.binary.version}</artifactId>
+<!--            <version>${kafka.version}</version>-->
+            <version>0.10.0-kafka-2.1.2</version>
```

Comment:
splice_machine is a core module, i.e. platform independent.
Kafka dependency is platform dependent. If we build against 0.10.0-kafka-2.1.2 but the run-time dependency is different and not fully compatible, we may run into obscure errors.

Reply:
Removed Kafka dependencies from splice_machine in commit [67b33b0](https://github.com/splicemachine/spliceengine/pull/3405/commits/67b33b00e537df5b9dfe5c4acc4ff7013ce1affa)